testsuite/features/step_definitions/command_steps.rb

```diff
@@ -789,7 +789,7 @@
     end
     cmd = "zypper mr --#{action} #{opt_repos} ||:;" unless opt_repos.empty?
     cmd += "zypper mr --#{action} #{mand_repos}" unless mand_repos.empty?
-  elsif os_family =~ /^centos/
+  elsif os_family =~ /^centos/ || os_family =~ /^rocky/
```

Comment:
yeah we would probably need to add more of these redhat-compatible systems here

Reply:
Yes but I would do that in a separate PR :wink: